src/apps/external_executor/executor_code_id.h

```diff
@@ -16,6 +16,12 @@ struct ExecutorNodeInfo
   ccf::Attestation attestation;
   std::vector<ccf::NewExecutor::EndpointKey> supported_endpoints;
 };
+using ExecutorId = ccf::EntityId<ccf::NodeIdFormatter>;
+typedef std::map<ExecutorId, ExecutorNodeInfo> ExecutorIDMap;
+typedef std::map<ExecutorId, crypto::Pem> ExecutorCertsMap;
+
+ExecutorIDMap executor_ids;
+ExecutorCertsMap executor_certs;
```

Comment:
Can we avoid globals here?

Reply:
Will keep this for now and will refactor later